### PR TITLE
For #8065 - Prelands strings for MR1

### PR DESCRIPTION
**Pull request by boek**
Prelands strings needed for MR1 as defined by https://docs.google.com/spreadsheets/d/1RMB_AGwzJjRbJ_ixye7hAOh1w3Vd0UWiy73zN_3nfd4/edit#gid=1681141951

### CODE REVIEW

File: Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -532,9 +550,11 @@ extension Strings {
     public static let AppMenuShowTabsTitleString = MZLocalizedString("Menu.ShowTabs.Title", tableName: "Menu", value: "Show Tabs", comment: "Label for the button, displayed in the menu, used to open the tabs tray")
     public static let AppMenuSharePageTitleString = MZLocalizedString("Menu.SharePageAction.Title", tableName: "Menu", value: "Share Page With…", comment: "Label for the button, displayed in the menu, used to open the share dialog.")
     public static let AppMenuCopyURLTitleString = MZLocalizedString("Menu.CopyAddress.Title", tableName: "Menu", value: "Copy Address", comment: "Label for the button, displayed in the menu, used to copy the page url to the clipboard.")
+    public static let AppMenuCopyLinkTitleString = MZLocalizedString("Menu.CopyLink.Title", tableName: "Menu", value: "Copy Link", comment: "Label for the button, displayed in the menu, used to copy the current page link to the clipboard.")
```
Comment (nbhasin2) [inline Client/Frontend/Strings.swift:553]: I thought we didn't wanna add tableName 😅

**Review by nbhasin2 — APPROVED** [review]
Thank you for landing this :)

File: Client/Frontend/Strings.swift (hunk 1 above)
Comment (boek) [reply Client/Frontend/Strings.swift:553]: @nbhasin2 😅 I was thinking about this. I think until we decide to get rid of the existing ones I thought I would continue keeping them grouped together to keep them consistent

Comment (nbhasin2) [reply Client/Frontend/Strings.swift:553]: Ah true make sense I am good with this @boek 🙂

File: Client/Frontend/Strings.swift [hunk 2]
```diff
@@ -61,6 +63,9 @@ extension Strings {
     public static let ShareContextMenuTitle = MZLocalizedString("HomePanel.ContextMenu.Share", value: "Share", comment: "The title for the Share context menu action for sites in Home Panels")
     public static let RemoveContextMenuTitle = MZLocalizedString("HomePanel.ContextMenu.Remove", value: "Remove", comment: "The title for the Remove context menu action for sites in Home Panels")
     public static let PinTopsiteActionTitle = MZLocalizedString("ActivityStream.ContextMenu.PinTopsite", value: "Pin to Top Sites", comment: "The title for the pinning a topsite action")
+    public static let PinTopsiteActionTitle2 = MZLocalizedString("ActivityStream.ContextMenu.PinTopsite2", value: "Pin", comment: "The title for the pinning a topsite action")
+    public static let UnpinTopsiteActionTitle2 = MZLocalizedString("ActivityStream.ContextMenu.UnpinTopsite", value: "Unpin", comment: "The title for the unpinning a topsite action")
+    public static let AddToShortcutsActionTitle = MZLocalizedString("ActivityStream.ContextMenu.AddToShortcuts", value: "Add to Shortcuts", comment: "The title for the pinning a shortcut action")
```
Comment (dnarcese) [inline Client/Frontend/Strings.swift:68]: I think we are missing "Remove from Shortcuts" string

**Pull request merged**